model/src/main/kotlin/PackageReference.kt

```diff
@@ -37,11 +37,16 @@ data class PackageReference(
          * The list of references to packages this package depends on. Note that this list depends on the scope in
          * which this package reference is used.
          */
-        val dependencies: SortedSet<PackageReference>,
+        // Do not serialize if empty to reduce the size of the result file.
```

Comment:
Maybe add: "There likely still are many cases where this is not empty, so it is obvious that dependencies are properly recognized." (Or something like that.)

Reply:
Both done.